### ISSUE TITLE
How to find the pretrained models to extract frame feature?

### ISSUE THREAD

**Issue by zhangyuygss** [olivesgatech/TA3N#9]
Hey, seems you use pretrained C3D or resnet to extract the frame feature,
is the model pretrained on Kinetics(C3D) and ImageNet(resnet)?
Where can I download the pretrained weight file?

By the way, why the backbone parts are not trained in the framework? To my knowledge, video classification
models (TSN, TRN) usually update these parts.

**Comment by cmhungsteve** [comment]
I used ResNet-101 pretrained on ImageNet.
The pretrained model is the built-in pytorch model, so it's easy to get it as long as you install pytorch.

End-to-end training is computation consuming, and I don't have computing resources as the authors of TSN, TRN, etc.
In addition, my focus is the temporal part, but the extracted feature vectors mainly encode spatial information.
If you have enough GPUs, you can try end-to-end training, and I think the accuracy would increase, but the training time will be much longer.
